### PR TITLE
Ports duplication improvements from tg

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

Closes https://github.com/yogstation13/Yogstation/pull/19797

https://github.com/tgstation/tgstation/pull/72572

# Changelog

No player facing changes

### CODE REVIEW

File: code/modules/holodeck/area_copy.dm [hunk 1]
```diff
@@ -1,59 +1,4 @@
...
-			thing.flags_1 |= HOLOGRAM_1
-		if(ismachinery(O))
-			var/obj/machinery/M = O
-			for(var/atom/contained_atom in M.component_parts)
-				contained_atom.flags_1 |= HOLOGRAM_1
-			if(M.circuit)
-				M.circuit.flags_1 |= HOLOGRAM_1
-	return O
-
+// DuplicateObject was renamed to duplicate_object and moved to code/__HELPERS/duplicating.dm
```
Comment (JohnFulpWillard) [inline code/modules/holodeck/area_copy.dm]: This comment is useless here, people looking for it will find duplicate_object and can search for that

**Review by JohnFulpWillard — APPROVED** [review]
!!

File: code/modules/holodeck/area_copy.dm (hunk 1 above)
Comment (ynot01) [reply code/modules/holodeck/area_copy.dm]: image

**Comment by Hislyna** [comment]
Oh thank god

File: code/modules/holodeck/area_copy.dm (hunk 1 above)
Comment (JohnFulpWillard) [reply code/modules/holodeck/area_copy.dm]: yuh huh remove it right friggin now

**Pull request merged**